apple/internal/resource_rules/apple_core_data_model.bzl

```diff
@@ -127,6 +136,12 @@ apple_core_data_model = rule(
             "swift_version": attr.string(
                 doc = "Target Swift version for generated classes.",
             ),
+            "outs": attr.output_list(
+                doc = """
+An optional list of expected output files. If not empty, the rule will
+return these files individually rather than returning a directory.
```

Comment:
Could we add some info on how users can figure out these paths? Could we make them shorter/easier to declare somehow?

Reply:
Yeah, I can definitely improve the docs. Shorter/easier paths are a bit trickier. It's complicated by the fact that the rule can accept multiple `xcdatamodeld` files, and generated source files are grouped under a different directory for each data model. If the rule accepted only one data model, this would be a lot simpler.

Perhaps we could make something like this work where the `outs` attr is a dict instead of a list:
```
apple_core_data_model(
    name = "explicit_outs_data_model",
    srcs = [
        "//test/starlark_tests/resources:swift_datamodel",
    ],
    outs = {
        "swift_datamodel": [
            "Item+CoreDataProperties.swift",
            "swift_datamodel+CoreDataModel.swift",
        ],
    },
    tags = common.fixture_tags,
)
```

Reply:
I went ahead and updated the implementation based on this idea. I also updated the docs. Let me know what you think.